### PR TITLE
Add transaction view for payment type

### PR DESCRIPTION
**Pull request by pavel-fokin**

### CODE REVIEW

File: utils/transaction.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import moment from "moment";
 
 const formatDateTime = (date) => {
-  return moment(date).utc().format("YYYY-MM-DD HH:mm:ss");
+  return moment(date).format("YYYY-MM-DD HH:mm:ss");
```
Comment (ihomp) [inline utils/transaction.js:4]: we have a function to show date and time
`import {fullDateAndTime } from '../utils/format'`
it shows the date and time in locally for user format

File: components/Transaction/Links.js [hunk 2]
```diff
@@ -1,22 +1,25 @@
-import Link from 'next/link'
+import Link from "next/link";
 
 const LinkExplorer = ({ hash }) => (
-    <a href={`https://bithomp.com/explorer/${hash}`} target="_blank" rel="noopener noreferrer">
-        {hash}
-    </a>
-)
+  <a
+    href={`https://bithomp.com/explorer/${hash}`}
```
Comment (ihomp) [inline components/Transaction/Links.js:5]: we need to replace old Explorer links to 
new links like 
/tx/
/nft/
/account/
etc

File: components/Transaction/TransactionDetails.js [hunk 3]
```diff
@@ -61,54 +39,61 @@ export const TransactionDetails = ({ tx }) => {
...
+              <Table.Data>
+                <LinkAccount address={tx.address} />
+              </Table.Data>
+            </Table.Row>
+            <Table.Row>
+              <Table.Data>Sequence:</Table.Data>
+              <Table.Data>#{tx.sequence}</Table.Data>
+            </Table.Row>
+            <Table.Row>
+              <Table.Data>XRLP fee:</Table.Data>
```
Comment (ihomp) [inline components/Transaction/TransactionDetails.js:85]: XRPL :)

File: components/Transaction/TransactionDetails.js [hunk 4]
```diff
@@ -61,54 +39,61 @@ export const TransactionDetails = ({ tx }) => {
...
+              <Table.Data>{tx.outcome?.fee}</Table.Data>
```
Comment (ihomp) [inline components/Transaction/TransactionDetails.js:86]:
```
import {amountFormat } from '../utils/format'

{amountFormat(tx.outcome?.fee)}
```

File: components/Transaction/TransactionPayment.js [hunk 5]
```diff
@@ -1,25 +1,26 @@
...
 
-    const source = tx.specification?.source?.address
-    const destination = tx.specification?.destination?.address
+export const TransactionPayment = ({ tx }) => {
+  const destination = tx.specification?.destination?.address;
 
-    return (
-        <TransactionDetails tx={tx}>
-            <tr>
-                <td>Source:</td>
```
Comment (ihomp) [inline components/Transaction/TransactionPayment.js:12]: Source was removed ? :)

File: utils/transaction.js (hunk 1 above)
Comment (pavel-fokin) [reply utils/transaction.js:4]: `fullDateAndTime` returns `Invalid Date` for `'2023-12-13T13:42:21.000Z'`. I'll localize timestamps a bit later. Let's keep as is for now.

File: components/Transaction/TransactionDetails.js (hunk 4 above)
Comment (pavel-fokin) [reply components/Transaction/TransactionDetails.js:86]: `amountFormat` gives `1.2e-11 XRP` for low values, it looks a bit not user friendly.

**Comment by pavel-fokin** [comment]
I fixed most of the comments. But I'll close this PR as it is included to the https://github.com/Bithomp/frontend-react/pull/94.

File: utils/transaction.js (hunk 1 above)
Comment (ihomp) [reply utils/transaction.js:4]: it expects the Unix timestamp I guess, so we rather update what endpoint returns from the backend, as we usually return timestamps in /v2/ 

Or as an option we can update the fullDateAndTime to accept the '2023-12-13T13:42:21.000Z' format too. 

You can leave it for now, but would be good to leave a comment so I won't forget to update later :)

**Pull request closed without merging**